### PR TITLE
Added a note #1559

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added  a note to missing before the closure of the issue #1559

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

File: OneDrive/enable-conditional-access.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ The following are known issues with this release:
      `Set-AdfsGlobalAuthenticationPolicy -PrimaryIntranetAuthenticationProvider @('WindowsAuthentication', 'FormsAuthentication')`
     
 - If you enable location-based conditional access, users will get a prompt about every 90 to 120 minutes by default when they leave the set of approved IP address ranges. The exact timing depends on the access token expiry duration (60 minutes by default), when their computer last obtained a new access token, and any specific conditional access timeouts put in place.
+
+- Silent account configuration won't work on devices for users who require multi-factor authentication.
```
Comment (JohanFreelancer9) [inline OneDrive/enable-conditional-access.md]:
```suggestion
- Silent account configuration won't work on devices of users who require multi-factor authentication.
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by kaarins** [comment]
This seems like a mistake. I don't think you mean to overwrite this article: https://docs.microsoft.com/en-us/onedrive/enable-conditional-access with https://docs.microsoft.com/en-us/onedrive/pre-provision-accounts

**Comment by j0rt3g4** [comment]
I think you're right I'd do the corrections @kaarins

**Pull request closed without merging**